README.md

```diff
@@ -76,6 +62,19 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Log in to API Mediation Layer and leverage base profile with token
+
+1. In your terminal, enter the `zowe auth login apiml` command.  
+2. Provide your username, password, host, and port for the API ML instance.  
+   A local base profile is created that contains your token.  
+3. In Zowe Explorer, hover over **DATA SETS**, **USS**, or **JOBS**.  
+4. Click the **+** icon.  
+5. Select **Create a New Connection to z/OS**.  
+6. Provide only a name for your profile and the base path for your API Mediation Layer instance.  
```

Comment:
This will not work in ZE. URL is still required (same with CLI). This zOSMF profile can only have this onformation if it is edited manually

Reply:
Aha, so here it should read "Provide a name, host, port, and the base path for your API Mediation Layer instance". Yes?

Reply:
no. Because the profile that will be created is an Optional Credential Profile. It will not use token. We are not handling creation in this area. Our assumption is that the user knows how to use base profiles.

Reply:
I think it would be best if the creation of zosmf  profile is through Zowe CLI as well. Or just use "Load the service profile that you are using with base profile" something like that

Reply:
Okay, I can change it